src/components/Todo-List.vue

```diff
@@ -1,21 +1,33 @@
 <template>
-  <div>
-    <p>I'm the todos component</p>
+  <div class="todo-list">
     <ul>
-      <li v-for="todo in todos">
-        <todo-item :todo="todo"></todo-item>            
+      <li v-for="todo in todos" class="todo-item">
+        <!-- <todo-item :todo="todo"></todo-item> -->
```

Comment:
Is there a particular reason why the todo-item component is no longer being used?